__tests__/integration/nanocontracts/bet.test.ts

```diff
@@ -481,14 +482,31 @@ describe('full cycle of bet nano contract', () => {
     expect(wallet.storage.processHistory.mock.calls.length).toBe(1);
   };
 
-  it('bet deposit', async () => {
-    await executeTests(hWallet);
+  it('bet deposit built in', async () => {
+    await executeTests(hWallet, builtInBlueprintId);
   });
 
   // The hathor-core and the wallet-lib are still not ready for
   // using nano contracts with a Multisig wallet
-  it.skip('bet deposit with multisig wallet', async () => {
-    await executeTests(mhWallet);
+  it.skip('bet deposit built in with multisig wallet', async () => {
+    await executeTests(mhWallet, builtInBlueprintId);
+  });
+
+  it('bet deposit on chain blueprint', async () => {
+    // For now the on chain blueprints needs a signature from a specific address
+    // so we must always generate the same seed
```

Comment:
Ok, so make it a constant

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-lib/pull/811/commits/0d01f9512097e12bbe7a9d00927d3d0dd86d1492